geode-core/src/main/java/org/apache/geode/internal/statistics/HostStatHelper.java

```diff
@@ -17,154 +17,64 @@
 import java.net.InetAddress;
 import java.net.UnknownHostException;
 
-import org.apache.geode.InternalGemFireException;
 import org.apache.geode.Statistics;
-import org.apache.geode.internal.PureJavaMode;
 import org.apache.geode.internal.net.SocketCreator;
 import org.apache.geode.internal.statistics.platform.LinuxProcFsStatistics;
 import org.apache.geode.internal.statistics.platform.LinuxProcessStats;
 import org.apache.geode.internal.statistics.platform.LinuxSystemStats;
-import org.apache.geode.internal.statistics.platform.OSXProcessStats;
-import org.apache.geode.internal.statistics.platform.OSXSystemStats;
 import org.apache.geode.internal.statistics.platform.OsStatisticsFactory;
 import org.apache.geode.internal.statistics.platform.ProcessStats;
-import org.apache.geode.internal.statistics.platform.SolarisProcessStats;
-import org.apache.geode.internal.statistics.platform.SolarisSystemStats;
-import org.apache.geode.internal.statistics.platform.WindowsProcessStats;
-import org.apache.geode.internal.statistics.platform.WindowsSystemStats;
 
 /**
  * Provides native methods which fetch operating system statistics.
```

Comment:
Add a better description of what this class is (it no longer has native methods).